core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -273,44 +275,56 @@ var StoryEditorPage = function() {
   };
```

Comment:
In `this.expectChaptersListToBe` above, can we check that chapter titles are visible before calling `.getText()` on them?